frontend/src/scenes/data-pipelines/batch-exports/BatchExportEditForm.tsx

```diff
@@ -455,14 +455,173 @@ export function BatchExportsEditFields({
+                            name="properties_data_type"
+                            label="Semi-structured data type"
+                            info={
+                                <>
+                                    Different PostHog models have semi-structured data fields in them, like
+                                    "events.properties". We can export these fields to Redshift as a "SUPER" type
+                                    column, or a "VARCHAR" column. We recommend "SUPER" over "VARCHAR" as "VARCHAR" has
+                                    a strict length limit that applies on the entire document, whereas with "SUPER" the
+                                    limit applies on each value in the document.
+                                </>
+                            }
+                        >
                             <LemonSelect
                                 options={[
                                     { value: 'varchar', label: 'VARCHAR(65535)' },
                                     { value: 'super', label: 'SUPER' },
                                 ]}
                             />
                         </LemonField>
+
+                        <LemonField
+                            name="mode"
+                            label="Command"
+                            className="flex-1"
+                            info={
+                                <>
+                                    Choose the SQL command used by the batch export. "COPY" has the best performance but
+                                    requires an S3 bucket we can connect to. "INSERT" performs worse but without any
+                                    additional requirements.
+                                </>
+                            }
+                        >
+                            <LemonSelect
+                                options={[
+                                    { value: 'COPY', label: 'COPY' },
+                                    { value: 'INSERT', label: 'INSERT' },
+                                ]}
+                            />
+                        </LemonField>
+
+                        {batchExportConfigForm.mode === 'COPY' && (
+                            <>
+                                <div className="flex gap-4">
+                                    <LemonField name="redshift_s3_bucket" label="S3 bucket name" className="flex-1">
+                                        <LemonInput placeholder="e.g. my-bucket" />
+                                    </LemonField>
+                                    <LemonField
+                                        name="redshift_s3_bucket_region_name"
+                                        label="S3 bucket region"
+                                        className="flex-1"
+                                    >
+                                        <LemonSelect
+                                            options={[
```

Comment:
nit: would be nice if we could consolidate this with the list of regions above (even nicer would be to pull this info from somewhere so we can ensure it remains up-to-date, but that's a bit of out scope :) )